doc/source/management/ssl-improvement.rst

```diff
@@ -0,0 +1,42 @@
+.. _percona-server.management.ssl:
+
+================================================================================
+SSL Improvements
+================================================================================
+
+|Persona Server| enables transmitting data in the encrypted form by using the
```

Comment:
This section interleaves SSL library choice and TLS version support. I'd make two separate sections. For TLS, link to the relevant bug